VisualStudioCode/README.md

```diff
@@ -18,3 +18,50 @@ After that, you can launch VS Code from your terminal:
 
 * `code .` will open VS Code in the current directory
 * `code myfile.txt` will open `myfile.txt` in VS Code
+
+### Python Extension
+
+Install this extension
+https://marketplace.visualstudio.com/items?itemName=ms-python.python
+
+Configure the two things below
+
+1. Change the default formatter to "Black" instead of "Autopep8". Very critical to avoid large diffs. Go to "Preferences" -> User Settings and update the setting "python.formatter.provider" to "Black" https://code.visualstudio.com/docs/python/editing
+
+2. Enable Format on Save Setting: "Editor: Format On Save" setting on Code -> Preferences -> Settings
+
+3. Open a python file and check if you write code that it automatically formats on saving the file.
+
+### Markdown rendering extension
+
+This extension allows you to see mardown files easily in Visual Studio itself so that we can read documentation directly from VS Code
```

Comment:
```suggestion
This extension allows you to see Markdown files easily in Visual Studio itself so that we can read documentation directly from VS Code
```